components/accordion/stories/accordion.stories.js

```diff
@@ -66,6 +66,7 @@ export default {
 		chromatic: { disableSnapshot: true },
 		componentVersion: version,
 	},
+	tags: ["!autodocs"],
```

Comment:
Since the `!autodocs` is being set at the story's default, do we need to keep the autodocs tag in the Compact and Spacious stories to turn them back on?

Reply:
(Ditto for a few other stories in this list with the same update)

Reply:
The ones with default `!autodocs` added like this use a custom MDX template that defines what stories are displayed (accordion already shows compact and spacious). I'm not sure if Storybook will fix it in the future, but right now it causes the error noted in the PR description if the components using a custom MDX were tagged with the default `autodocs`.